### ISSUE TITLE
although I was to use the learning rate of 1*10e-8, but the value of loss is very big

### ISSUE THREAD

**Issue by cchenzhou** [s9xie/hed#29]
Hi,
First of all thank you provide the project code generously. I'am trying to train the network on BSDS dataset. But the loss can not convergence. Then, I can understand the value of max_iter equals 30001,and the training process does't  stop until iteration equals 100000. 
Any idea about how to fix the error?
Thank you!
chen

**Comment by mhl20110088** [comment]
I have encountered the same problem,but I do not know how to modify it.I think ,you try to  modify the loss layer of the network .

**Comment by mengzhangjian** [comment]
I also encounter the same problem , the initial loss is very large and does not converge ,anyone knows it ?

**Comment by chaobeiying** [comment]
Have you solved your problem?
